lib/version_utils.pm

```diff
@@ -402,10 +509,16 @@ sub has_license_on_welcome_screen {
         || (is_sle('=15') && is_s390x()));
 }
 
+=head2 has_license_to_accept
+
```

Comment:
For this and others where the documentation is non existent: Please give a brief description, even if it's just too obvious